outputs.tf

```
@@ -12,3 +12,8 @@ output "aws_kms_key_arn" {
   description = "The Amazon Resource Name (ARN) of the key."
   value       = aws_kms_key.s3.arn
 }
+
+output "aws_kms_key_id" {
+  description = "The Amazon Resource Name (ARN) of the key."
```

Sorry, set the description here and then update it in the readme with `terraform-docs` using  pre-commit.

```suggestion
  description = "The globally unique identifier for the key."
```